src/providers/proxy/proxy_init.c

```diff
@@ -76,6 +76,28 @@ static errno_t proxy_id_conf(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static bool local_auth_enabled(struct be_ctx *be_ctx)
+{
+    int ret;
+    char *local_policy = NULL;
+    bool res;
+
+    ret = confdb_get_string(be_ctx->cdb, NULL, be_ctx->conf_path,
+                            CONFDB_DOMAIN_LOCAL_AUTH_POLICY,
+                            "match", &local_policy);
```

Comment:
```
$ grep -rn \"match\" *
responder/pam/pamsrv_cmd.c:943:                            "match", &local_policy);
responder/pam/pamsrv_cmd.c:954:    } else if (strcasecmp(local_policy, "match") == 0) {
responder/pam/pamsrv_cmd.c:1232:              && strcasecmp(local_policy, "match") == 0) {
```
  --  "match" begs for a define, imo...

Reply:
fixed

Reply:
I meant in the entire code base, not in 'proxy provider' code locally...
But it can be done in another PR.